stories/Components/UIcomponents/Cards/FeaturedCards/featuredContentCard.scss

```diff
@@ -92,31 +148,47 @@
       background-color: $color-yellow;
 
       &::before {
-        background: transparent linear-gradient(207deg, hsla(0, 0%, 100%, 0.2) 30%, rgba(255, 235, 0, 0.8));
+        background:
+          transparent
+          linear-gradient(207deg,
+          hsla(0, 0%, 100%, 0.2) 30%,
+          rgba(255, 235, 0, 0.8));
```

Comment:
@pallavikadam-srijan Can you use any existing color variables here? If not define these in the global color variable scss and re-use it here rather than re-writing in multiple places so that it can be maintained.